packages/@coorpacademy-components/src/molecule/card-content/index.js

```diff
@@ -170,12 +182,13 @@ const ContentInfo = ({
             <ContentBadge category={badgeCategory} label={badgeLabel} />
           ) : null}
         </div>
-        <CardTitle title={title} empty={empty} courseContent={courseContent} />
+        <CardTitle title={title} empty={empty} courseContent={courseContent} theme={theme} />
```

Comment:
```suggestion
        <CardTitle title={title} empty={empty} courseContent={courseContent} />
```
pas besoin de theme l'élément root a déjà la class coorpmanager